### DISCUSSION TITLE
serverless deploy fail on big sur 11.3.1

### DISCUSSION THREAD

**Discussion by alepru** [serverless/serverless#10089] [Q&A]
Hi to all,

when i try to deploy my serverless application on a macbook pro 16 with big sur 11.3.1 i receive this error:

`Serverless: Uploading CloudFormation file to S3...
Serverless: Recoverable error occurred (socket hang up), sleeping for ~6 seconds. Try 1 of 4
Serverless: Recoverable error occurred (read ECONNRESET), sleeping for ~5 seconds. Try 2 of 4
Serverless: Recoverable error occurred (socket hang up), sleeping for ~4 seconds. Try 3 of 4
Serverless: Recoverable error occurred (read ECONNRESET), sleeping for ~6 seconds. Try 4 of 4
 
 Serverless Error ----------------------------------------
 
  write EPIPE
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              14.17.6
     Framework Version:         2.57.0
     Plugin Version:            5.4.5
     SDK Version:               4.3.0
     Components Version:        3.17.1`

i have tried different version of serverless ( also the last )  but the error it's the same.

on my windows 7 the deploy works perfectly.

i've found other users with the same problem:
https://community.jamf.com/t5/jamf-pro/mac-updates-broke-serverless-com-deployments-artifact-uploads-to/td-p/237162.

thanks.

**Comment by medikoo** [comment]
@alepru `ECONNRESET` error signal problems with network connection, we're planning to improve reporting of that (see https://github.com/serverless/serverless/issues/7733) so the message is more meaningful.

I'm moving to discussions, as it doesn't report the issue with Framework itself
